torch/export/graph_signature.py

```diff
@@ -487,6 +496,11 @@ def _(old, new, user):
 
         return _
 
+    def __str__(self):
```

Comment:
nit: Could we add the word `GraphSignature` here? So that it looks like:
```
Graph Signature:
    # Inputs
    b_buf: kind=BUFFER, fqn='buf'

    # Outputs
    add_1: BUFFER_MUTATION
```

Then when we print the exported program we won't need to add the "GraphSignature" word

Reply:
I considered it, but trying to be consistent with above and below here. Above, graph module is printed without prefix and adjusted for indent, and we add "ExportedProgram:". We follow that pattern here. Now, arguably one might say that prefix applies to everything and not just the graph module, so not really a pattern, so I also looked for a hint below, where "Range constraints:" is another thing we print. 

Besides, I wasn't sure whether we want to say `ExportGraphSignature` or just graph signature if we move this inside...